cogito/gchatsink.go

```diff
@@ -3,6 +3,7 @@ package cogito
 import (
 	"context"
 	"fmt"
+	"io/fs"
```

Comment:
Like afero FS, but in the Go stdlib (makes sense only for read-only FS, like the one used by Cogito)